src/ui-config/reservePatches.ts

```diff
@@ -35,6 +35,8 @@ export const SYMBOL_MAP: { [key: string]: string } = {
   'AAVE.e': 'AAVE',
   'USDT.e': 'USDT',
   'USDC.e': 'USDC',
+  'FRAX.e': 'FRAX',
```

Comment:
symbol of Frax on avalanche is FRAX https://snowtrace.io/address/0xD24C2Ad096400B6FBcd2ad8B24E7acBc21A1da64#readContract

symbol of Mai is MAI on avalanche: https://snowtrace.io/address/0x5c49b268c9841aff1cc3b0a418ff5c3442ee3f3b#readContract

Reply:
ok, deleted it
reinsured just in case =)